WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostCardActionsHandler.kt

```diff
@@ -102,8 +110,20 @@ class ReaderPostCardActionsHandler @Inject constructor(
         AppLog.d(AppLog.T.READER, "Block site not implemented")
     }
 
-    private fun handleLikeClicked(postId: Long, blogId: Long) {
-        AppLog.d(AppLog.T.READER, "Like not implemented")
+    private fun handleLikeClicked(post: ReaderPost) {
```

Comment:
Not sure if this all should be happening at this location.

Reply:
I think we can either handle it here or return SnackbarMessageHolder instead of ReaderRepositoryCommunication. Imo both approaches are fine.

Reply:
I am partial to leaving it as is, because if we want to notify the UI that it's started we can. Plus it feels kind of odd that the use case knows that we want to set a Snackbar message a.k.a. a UI element. But then I guess you can say the same thing about the action handler, so what do I know 🤷 .

Reply:
>Plus it feels kind of odd that the use case knows that we want to set a Snackbar message a.k.a. a UI element.

yep, I kind of agree.

>But then I guess you can say the same thing about the action handler, so what do I know 🤷 .

The handler directly handles ui interactions so it feels ok to me that it works with UI elements.